src/invariant/LiabilitiesMatchOffersTests.cpp

```diff
@@ -295,27 +300,22 @@ TEST_CASE("Invariant for liabilities", "[invariant][liabilitiesmatchoffers]")
     SECTION("create then modify then delete offer")
     {
         auto offer = generateOffer(cur1, cur2, 100, Price{1, 1});
-        auto selling = generateSellingLiabilities(lm, offer, false, true);
-        auto buying = generateBuyingLiabilities(lm, offer, false, true);
+        auto selling = generateSellingLiabilities(*app, offer, false, true);
```

Comment:
line 288 `LedgerManager& lm = app->getLedgerManager();`
'lm': local variable is initialized but not referenced